app/controllers/application_controller.rb

```diff
@@ -67,6 +70,15 @@ def timeout_timer
 
 private
 
+  # @return [Boolean]
+  def maintenance?
+    Rails.application.maintenance?
+  end
+
+  def service_unavailable_page
+    redirect_to static_path('service-unavailable') unless request.path == '/service-unavailable'
```

Comment:
Is it not possible to check the path in the predicate?

Reply:
[37aa9b6](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/941/commits/37aa9b6ee52227bd8b6d9dd34c555eab44b37802)